core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -227,100 +226,76 @@
               (mouseleave)="closeSubmenuIfNotMobile($event)">
             <div class="d-flex flex-column">
               <div class="top-section d-flex">
-                <a class="item item-border d-flex flex-column"
-                   (keydown)="onMenuKeypress($event, 'getInvolvedMenu', {shiftTab: ACTION_CLOSE, enter: ACTION_CLOSE})"
-                   href="/partnerships"
-                   [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.PARTNERSHIPS.ROUTE"
-                   rel="noopener">
+                <div class="item item-border d-flex flex-column innvolved-item" onclick="location.href='/partnerships'"
```

Comment:
I believe it should be (click) instead of onclick? Does this work as expected?

Reply:
(click) doesn't seem to work here

Reply:
We prefer using (click) over onclick and performing the redirection using a function which is added to component file.

Reply:
Done!

Reply:
Thanks @Lawful2002 .